egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -20,14 +20,14 @@ def GetArgs():
                                                  "--random-seed 1 data/train data/train_rvb",
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument("--rir-list-file", type=str, required = True, 
+    parser.add_argument("--rir-list-file", type=str, action='append', required = True, dest = "rir_list_file_array", 
```

Comment:
When you want to combine two different lists with different number of items it would be preferable to assign different probability masses to each of these lists. 

e.g. real RIR list might have 500 RIRs and the sim. RIR list might have 20000 RIRs. Even in this case we would like to sample from real RIR list with 0.5 probability and sim RIR with 0.5 probability as we might believe real RIRs are more helpful. 
or 
Imagine the case where we know that test data is more likely to occur from a particular type of rooms in this case we would want to sample more RIRs from similar rooms even in training data. Assuming we can separate these RIRs into a new list we would like to assign a higher probability mass to this RIRs in this list.

So I would recommend supporting assigning different probability masses to different lists.